flixel/input/mouse/FlxMouse.hx

```diff
@@ -17,11 +17,11 @@ import flixel.system.FlxAssets;
 import flixel.system.replay.MouseRecord;
 import flixel.util.FlxDestroyUtil;
 #if FLX_NATIVE_CURSOR
-import flash.Vector;
-import flash.geom.Matrix;
-import flash.geom.Point;
-import flash.ui.MouseCursor;
-import flash.ui.MouseCursorData;
+import openfl.Vector;
+import openfl.geom.Matrix;
+import openfl.geom.Point;
+import openfl.ui.MouseCursor;
+import openfl.ui.MouseCursorData;
```

Comment:
looks like flash is failing here because there is no `openfl.ui.MouseCursorData`

Reply:
https://github.com/HaxeFlixel/flixel/blob/dev/flixel/system/macros/FlxDefines.hx#L144
I think FLX_NATIVE_CURSOR is only used with flash, so lets just use `flash.ui.MouseCursorData`

Reply:
ok?

Reply:
Nah